src/demo/QueryContainer.tsx

```diff
@@ -24,23 +24,66 @@
 
 import React from "react"
 import { ILook } from "@looker/sdk"
+import {
+  Heading,
+  Box,
+  Table,
+  TableHead,
+  TableRow,
+  TableHeaderCell,
+  Text,
+  TableDataCell,
+  TableBody
+} from "@looker/components"
```

Comment:
Is there a way to import each one individually to reduce package size when using `yarn start`? It would be lame if the only time we have an optimized package is when we build.

(I may be totally misunderstanding how imports work here, feel free to educate me)

Reply:
Any word on this?